docs/source/python/data.rst

```diff
@@ -264,6 +270,32 @@ individual arrays, and no copy is involved:
    arr.type
    arr
 
+Map arrays
+~~~~~~~~~~
+
+MapArrays can be constructed from lists of lists of tuples (key-item pairs), but only if
```

Comment:
```suggestion
Map arrays can be constructed from lists of lists of tuples (key-item pairs), but only if
```

? We're not really consistent, but in the sections above (and also the "Map arrays" title) we typically use the prose "struct array" and not the class name "StructArray"